### ISSUE TITLE
[Request] Fisheye camera model

### ISSUE THREAD

**Issue by eymengunay** [justadudewhohacks/opencv4nodejs#197]
Hello @justadudewhohacks,

I have noticed that [Fisheye camera model](https://docs.opencv.org/trunk/db/d58/group__calib3d__fisheye.html) is currently not supported. (I went through the API documentation but I'm not sure if I missed it somehow.)

Do you have any plans on adding this feature as well?

Thank you!

**Comment by justadudewhohacks** [comment]
Hi,

Yep, it's not implemented. Added to the todo list, but not sure how soon I will find the time to add it.

**Comment by eymengunay** [comment]
👍Thank you @justadudewhohacks for your prompt answer.

**Comment by justadudewhohacks** [comment]
Just reopening, to keep track of this.

**Comment by justadudewhohacks** [comment]
Nope, still waiting for contributors. Happy to take PRs for any kind of new bindings.
